example/basic/main.go

```diff
@@ -1,156 +1,53 @@
+// This is an example program that makes an HTTP request and prints response
```

Comment:
This is fresh new code for a "basic" example. The same code is used in the repo's README.md file.